### ISSUE TITLE
Better error reporting for `dune-release publish`

### ISSUE THREAD

**Issue by pitag-ha** [ocamllabs/dune-release#252]
When `dune-release publish` fails due to wrong global config data (in my case, I had copied the path to my opam-repository clone wrongly), we should let the user know what happened and how to fix it: through `dune-release config`.

**Comment by lindig** [comment]
Is there a `dune-relase config` command? Not in my version 1.3.3.

**Comment by gpetiot** [comment]
Hi, it has been added in [dune-release.1.4.0](https://github.com/ocamllabs/dune-release/releases/tag/1.4.0), released this week.

**Comment by gpetiot** [comment]
@pitag-ha did you ocpy the output trace by any chance?

**Comment by pitag-ha** [comment]
No, I haven't. That's a good point, though... I'll do it next time when I run into something I want to file an issue for. Also, it will be easy for me to reproduce, so I can do that tomorrow morning and send the output trace.

**Comment by pitag-ha** [comment]
I didn't remember well what I had in mind could be improved, when I wrote this issue. Sorry for that! There are two things I need to correct:
1. it's about `dune-release submit`,  not about `dune-release publish`.
2. the information given about the error is perfect (it was just confusing to me at the moment why it said it wasn't a valid Git repository, since I was just missing a `/` at the end of the path):
```
[-] Submitting 
[-] Preparing pull request to ocaml/opam-repository
dune-release: [ERROR] /home/sonja/Documents/Tarides/opam-repository is not a valid Git repository.
```
what I think might be nice adding is that now it can be fixed via 'opam config'.

I've corrected the issue message itself.
